docker/complement/conf/start_for_complement.sh

```diff
@@ -104,6 +104,8 @@ openssl x509 -in /conf/server.tls.crt -noout -text | grep DNS:
 export SYNAPSE_TLS_CERT=/conf/server.tls.crt
 export SYNAPSE_TLS_KEY=/conf/server.tls.key
 
+pip install -U setuptools
```

Comment:
Can you add a comment explaining why we're doing this please?

Reply:
I think that is unnecessary now, will remove and see.